api/devices/schema.py

```diff
@@ -48,16 +57,26 @@ def mutate(self, info, **kwargs):
 
 
 class UpdateDevice(graphene.Mutation):
+    """
+         Returns the device payload after updating
+    """
     class Arguments:
-        device_id = graphene.Int()
-        name = graphene.String()
-        device_type = graphene.String()
-        location = graphene.String()
-        room_id = graphene.Int(required=True)
+        device_id = graphene.Int(
+            description="Unique identifier of the tag")
+        name = graphene.String(
+            description="The name field of the device")
+        device_type = graphene.String(
+            description="The type of device field")
+        location = graphene.String(
+            description="The location field where the device is found")
+        room_id = graphene.Int(
+            required=True,
+            description="The room id where the device is located")
     device = graphene.Field(Devices)
 
     @Auth.user_roles('Admin')
     def mutate(self, info, device_id, **kwargs):
+        # mutation to update a device
```

Comment:
Hello @TeamoreA , I have realized to you have added this kind of comment in several methods? Is there a reason as to why? If so please kindly let us know.

Reply:
There was kind of repetition and I discovered they were not even appearing on the docs so I have removed them already